code/modules/antagonists/clockcult/clock_helpers/clock_rites.dm

```diff
@@ -196,4 +196,66 @@
 	CL.visible_message("<span class='warning'>[CL] materialises out of thin air!")
 	new /obj/effect/temp_visual/ratvar/sigil/transmission(T,2)
 
+//summons a soul vessel, which is the clockwork cult version of a soul shard. It acts like a posibrain and, as long as the target has a brain, a soul shard.
+/datum/clockwork_rite/soul_vessel
+	name = "Rite of the Vessel" //The name of the rite
+	desc = "This rite is used to summon a soul vessel, a special posibrain that makes whoever has their brain put into it loyal to the Justiciar.,\
+	 When put into a cyborg shell, the created cyborg will automatically be a servant of Ratvar."
+	required_ingredients = list(/obj/item/stack/cable_coil, /obj/item/stock_parts/cell/, /obj/item/organ/cyberimp)
+	power_cost = 2500 //These things are pretty strong, I won't lie
+	requires_full_power = TRUE
+	cast_time = 50
+	limit = INFINITE
+	rite_cast_sound = 'sound/magic/summon_guns.ogg'
+
+/datum/clockwork_rite/soul_vessel/cast(var/mob/living/invoker, var/turf/T, var/mob/living/carbon/human/target)
+	. = ..()
+	if(!.)
+		return FALSE
+	var/obj/item/mmi/posibrain/soul_vessel/SV = new /obj/item/mmi/posibrain/soul_vessel(T)
+	SV.visible_message("<span class='warning'>[SV] materalizes out of thin air!")
+	new /obj/effect/temp_visual/ratvar/sigil/transmission(T,2)
+
+
+/datum/clockwork_rite/cyborg_transform
+	name = "Rite of the Divine Form"
+	desc = "This rite is used to ascend into a cyborg, gaining unique scripture and a loadout that depends on which module is chosen. Consult the wiki for details on each cyborg module's loadout. Mutually exclusive to Enhanced Form."
+	required_ingredients = list(/obj/item/mmi/posibrain, /obj/item/stack/cable_coil, /obj/item/stock_parts/cell/super, /obj/item/bodypart/l_arm/robot, /obj/item/bodypart/r_arm/robot, /obj/item/bodypart/l_leg/robot, /obj/item/bodypart/r_leg/robot, /obj/item/bodypart/chest/robot, /obj/item/bodypart/head/robot, /obj/item/bodypart/r_leg/robot, /obj/item/bodypart/l_leg/robot)
+	power_cost = 20000
+	requires_human = TRUE
+	requires_full_power = FALSE
+	cast_time = 100
+	limit = INFINITE
+	rite_cast_sound = 'sound/magic/disable_tech.ogg'
+
+/datum/clockwork_rite/cyborg_transform/cast(var/mob/living/invoker, var/turf/T, var/mob/living/carbon/human/target)
+	. = ..()
+	if(!.)
+		return FALSE
+	target.visible_message("<span class='warning'>The robotic parts magnetize to [target], the new frame's eyes glowing in a brilliant yellow!")
+	var/mob/living/silicon/robot/R = target.Robotize()
+	R.cell = new /obj/item/stock_parts/cell/upgraded/plus(R)
+	new /obj/effect/temp_visual/ratvar/sigil/transmission(T,2)
+
+/datum/clockwork_rite/golem_transform
+	name = "Rite of the Enhanced Form"
+	desc = "This rite is used to shed one's flesh to become a clockwork automaton, becoming immune to many environmental hazards as well as being more resilient to incoming damage. Mutually exclusive to Divine Form."
```

Comment:
Becoming a scrap golem doesn't look like it prevents the cyborg_transform rite, though it sounds like such would be intended when it's called "Mutually exclusive"?